TipCatDotNet.Api/Options/StripeOptions.cs

```diff
@@ -0,0 +1,9 @@
+namespace TipCatDotNet.Api.Options
+{
+    public class StripeOptions
+    {
+        public string PublishableKey { get; set; } = string.Empty;
```

Comment:
Options are requires, so better leave them non-initiated and without defaults to catch configuration errors fast